lib/l10n/arb/app_en.arb

```diff
@@ -118,8 +118,8 @@
     "loadingTitle": "Please wait...",
     "loadingMessage": "Please wait while we are loading your data...",
     "buttonChange": "Change",
-    "welcomeContinue": "Let's go",
-    "enterEmail": "Let's go",
+    "welcomeContinue": "Let’s begin",
```

Comment:
Do we need a poEditor PR for this?

Reply:
this is from poeditor, i merged the Poeditor branch into the feature branch to keep it at 1 PR